Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/ClearPass.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Memory/SystemAllocator.h>
+#include <Atom/RPI.Public/Pass/RenderPass.h>
+
+namespace AZ
+{
+    namespace RPI
+    {
+        //! A simple pass to clear a render target
+        class ClearPass
```

Comment:
This is a bad idea. You are basically saying that its ok to create a full pass just to do a clear and it will encourage people to use this class which wrong. 

Ideally clear load action should be part of the pass that needs to use the resource. But in this case based on our conversations can we not just move the clear pass to the parent pass related to the msaa resource we want to clear?

Reply:
At the very least this should be called IneffecientClearPass and there needs to comment saying to NOT use class this and to be only used for very specific reasons.

Reply:
+1 on that as per my general remark